src/kademlia/dht_tracker.cpp

```diff
@@ -238,53 +238,10 @@ namespace libtorrent { namespace dht
 		m_dht.announce(ih, listen_port, flags, f);
 	}
 
-	namespace {
-
-	// these functions provide a slightly higher level
-	// interface to the get/put functionality in the DHT
-	bool get_immutable_item_callback(item& it, boost::function<void(item const&)> f)
```

Comment:
I'm relying on these wrappers to implement IPv4/IPv6 dual stack operations so they had better stick around.